index.js

```diff
@@ -1,5 +1,51 @@
 const core = require('@actions/core');
 const aws = require('aws-sdk');
+const assert = require('assert');
+
+// The max time that a GitHub action is allowed to run is 6 hours.
+// That seems like a reasonable default to use if no role duration is defined.
+const MAX_ACTION_RUNTIME = 6 * 3600;
+
+async function assumeRole(params) {
+  // Assume a role to get short-lived credentials using longer-lived credentials.
+  const isDefined = i => !!i;
+
+  const {roleToAssume, roleDurationSeconds, accessKeyId, secretAccessKey, sessionToken, region} = params;
+  assert(
+      [roleToAssume, roleDurationSeconds, accessKeyId, secretAccessKey, region].every(isDefined),
+      "Missing required input."
```

Comment:
Would be helpful in the error message to at least specify that this is for the assume role call, since some of these are not required input for the action overall.

Reply:
Good point. Adding.